charts/prometheus-smartctl-exporter/Chart.yaml

```
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.8.0
+version: 0.8.1
```

Let us make it a minor release rather than a patch, please.
```suggestion
version: 0.9.0
```